src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -37,12 +37,38 @@
 from clusterfuzz._internal.system import environment
 
 
+def _maybe_clear_progression_last_min_max_metadata(testcase, uworker_output):
+  """Clears last_progression_min and last_progression_max when 
+  clear_min_max_metadata is set to True"""
+  task_output = uworker_output.progression_task_output
+  if task_output is None:
+    return
+
+  if task_output.clear_min_max_metadata:
+    testcase.delete_metadata('last_progression_min', update_testcase=False)
+    testcase.delete_metadata('last_progression_max', update_testcase=False)
+
+
+def _save_current_fixed_range_indices(testcase, uworker_output):
```

Comment:
Try not to move functions around like this please, it makes the diff harder to read.